conjure-java-jaxrs-client/src/test/java/com/palantir/conjure/java/client/jaxrs/JaxRsClientDialogueEndpointTest.java

```diff
@@ -177,6 +178,79 @@ public void testUnsupportedHttpMethod_arbitrary() {
                 .hasMessageContaining("ARBITRARY");
     }
 
+    @Test
+    public void testTrailingWildcardParameter_slashes() {
+        ConjureRuntime runtime = DefaultConjureRuntime.builder().build();
+        Channel channel = mock(Channel.class);
```

Comment:
its probably cleaner to create the mocks at the class level instead of inside almost every test

Reply:
I find it difficult to use tests that configure mocks at the class level :-/
They make it more difficult to add tests for the same functionality with slightly different structure, and make it easy to accidentally invalidate existing tests while changing the common setup.

Reply:
Sorry, I was just suggesting creating the mocks i.e. `@Mock Channel channel` not configuring the responses. As @iamdanfox suggests we should factor out setup logic